### PR TITLE
Added Anthem Prep

### PR DESCRIPTION
**Pull request by apollyon600**

### CODE REVIEW

**Comment by philipto** [comment]
@apollyon600 Please provide us with:
- the school official website URL
- the school street address
- an URL of a page on the official website where the school offers an IT-related long-term (>1 year) course
- a proof, which shows that the school recognizes the domain you are submitting as an official email domain for the students.

**Comment by apollyon600** [comment]
- https://anthemprep.greatheartsamerica.org
- 39808 N Gavilan Peak Pkwy, Anthem, AZ 85086
- Logic and Coding is not listed in the curriculum, but it is offered to the Seniors, if you'd like you can email them asking if Logic and Coding exists. https://anthemprep.greatheartsamerica.org/academics/curriculum/
- There isn't really much I can provide, a few minor evidence I can provide is the following:

People who are registered in the school have a school specified email, it being
FIRSTNAME + . + LASTNAME + 2 DIGIT YEAR NUMBER THEY WERE BORN + @ghazanthemprep.org
image
*Some contents were blurred for privacy reasons*

Another is logging into google classroom, it says that this account is managed by ghazanthemprep.org
image

Basically it, hope it is enough!

**Comment by philipto** [comment]
@apollyon600 Thank you for the clarification. Pull request merged.

**Comment by philipto** [comment]
This domain is abused and involved in the fraud. It is removed from the repository.
